cypress/integration/home/home.spec.js

```diff
@@ -0,0 +1,9 @@
+describe("sees home", () => {
```

Comment:
Any specific reason for keeping the file structure this way?
`cypress/integration/home...`?

As our unit tests are in this way
`./tests/unti/...`
so doesn't it makes sense to keep integration tests in the same way?
`./tests/integration/...`

Reply:
No specific reason. That's how cypress created directories when I installed it and I went with default. Let me know if you think there can be a different structure. We can easily control it.

Reply:
I agree with Deepansh here too. Let's keep the structure that Deepansh proposed.

Reply:
Done.